web/src/app/components/LoginComponents/UsernameForm.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useRef, useState } from "react";
+import LogoV1 from "image/LogoV1.svg"
+import image1 from "image/image1.svg"
+import Image from "next/image";
+import { useRouter } from 'next/navigation'
+
+
+export default function UsernameForm() {
+
+    const router = useRouter();
+
+    const [inputValue, setInputValue] = useState('');
+    const usernameRef = useRef<HTMLButtonElement>(null);
+    const [changedUsername, setChangedUsername] = useState(false);
+
+    const changeUsername = () => {
+        
+        if (usernameRef.current) {
+            if (inputValue.trim().length < 3) {
+
+                alert('Your username must be a minium of 3 characters')
```

Comment:
This will be upgraded to a notification when we have it that PR merged into main. So I'm just using alert as a placeholder for now.